pkg/release/release.go

```diff
@@ -554,18 +562,15 @@ func (r *Release) test(client helm.Client, hr *apiV1.HelmRelease) (err error) {
 	defer func(start time.Time) {
 		ObserveReleaseAction(start, TestAction, err == nil, hr.GetTargetNamespace(), hr.GetReleaseName())
 	}(time.Now())
-	status.SetStatusPhase(r.hrClient.HelmReleases(hr.Namespace), hr, apiV1.HelmReleasePhaseTesting)
```

Comment:
is there an advantage to moving these 3 test-related `SetStatusPhase` calls outside the `test()` method? Elsewhere we are keeping these calls inside the action method e.g. with `install()`, `upgrade()`, `rollback()`, which seems useful in case the method needs to be called in multiple places which may be a possibility with e.g. #452.

Reply:
it was required but not required anymore